test/controllers/contentProperty.controller.test.js

```diff
@@ -506,10 +508,33 @@ describe('Content Property Controller', () => {
       });
     });
 
-    context('when the areaId parameter is given only', () => {
+    context('when areas are sent in different cases', () => {
+      const areaNameInDifferentCases = ['Ikoyi', 'IKOYI', 'IKOyi', 'ikoyi'];
+      [...new Array(4)].map((_, index) =>
```

Comment:
This is more preferable:

```suggestion
     areaNameInDifferentCases.map((_, index) =>
```